src/test/kotlin/org/koil/image/PersistedImageServiceTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.koil.image
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotNull
+import org.junit.jupiter.api.Test
+import org.koil.BaseIntegrationTest
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.mock.web.MockMultipartFile
+
+class PersistedProfileImageServiceTest : BaseIntegrationTest() {
+
+    @Autowired
+    lateinit var profileImageService: ProfileImageService
+
+    val example = MockMultipartFile(
+        "large.jpg",
+        "large.jpg",
+        "IMAGE/JPEG",
+        this::class.java.getResourceAsStream("/img/large.jpg")!!
+    )
+    val alternative = MockMultipartFile(
+        "large.png",
+        "large.png",
+        "IMAGE/PNG",
+        this::class.java.getResourceAsStream("/img/large.png")!!
+    )
+    val exampleScaled = this::class.java.getResourceAsStream("/img/scaled-jpg.png")!!
```

Comment:
We don't use this, perhaps it was intended for a test where we scale down large.jpg?